drivers/sensor/amg88xx/amg88xx.h

```diff
@@ -38,7 +38,7 @@
 #define AMG88XX_OUTPUT_BASE	0x80 /* Base address for the output values */
 
 #define AMG88XX_PCLT_NORMAL_MODE	0x00
-#define AMG88XX_PCLT_SLEEEP_MODE	0x10
+#define AMG88XX_PCLT_SLEEP_MODE	0x10
```

Comment:
This seems to change the alignment of the `0x10` value so that it's no longer aligned with the other values above and below it (probably one more tab needed)

Reply:
fixed, thanks!